website/content/en/_index.md

```diff
@@ -31,8 +31,8 @@ Today there are two active working groups under the TAG -
 
 ## Meetings
 
-Every two weeks on Wednesday at 16:00 UTC ([convert to your local
-time](https://dateful.com/convert/utc?t=16)).
+Every two weeks on Wednesday at 11:00am EDT ([convert to your local
+time](https://www.worldtimebuddy.com/?qm=1&lid=100,5,8,12&h=100&date=2024-5-30&sln=15-16&hf=1)).
```

Comment:
Thanks for making these updates! Just curious why you switched to world time buddy?

The URL includes a date which will be difficult to maintain and specifically will cause issue during day light savings changes.

I believe dateful can work as of "today" on click which may be a better long term fit?

E.g.
https://dateful.com/convert/est-edt-eastern-time?t=11

Reply:
Of course! I switched (maybe naively) because of miss understanding of how dateful worked and being able to visually see the timezone across the areas we were trying to advertise.

Upon review, I agree its probably better to keep dateful but update it to use EDT as the source time.